### ISSUE TITLE
Macro and /click LM_Bn broken in 10.0

### ISSUE THREAD

**Issue by mbattersby** [xod-wow/LiteMount#113]
Blizzard have broken /click on SecureActionButton with WoW 10.0. It was a bug that was fixed, but is now a deliberate breakage to support hold-to-cast.

See this issue: https://github.com/Stanzilla/WoWUIBugs/issues/317 (edit)

We are waiting on a fix. There is no global work-around, but you can half-fix it.

Run this:
```
/dump GetCVar("ActionButtonUseKeyDown")
```
It will print a value into your first chat tab.

If it looks like this `[1]="1"` then change your macro to `/click LM_B1 LeftButton 1`.

If it looks like this `[1]="0"` then make sure your macro has no button argument: `/click LM_B1` or is `/click LM_B1 LeftButton 0`

**Comment by mbattersby** [comment]
It's still broken. This might work, not sure. Just waiting for Blizzard really, it's part of the same problem that stops actionbar addons from supporting hold-to-cast so I assume it will be resolved some day.

`/click LM_B1 LeftButton 1`

**Comment by sariouso** [comment]
> 

Thank you. Adding the 1 fixed it.

**Comment by rodalpho** [comment]
Yes, the 1 fixed it. Thanks!

**Comment by sigprof** [comment]
Apparently `/click LM_B1 L 1` also works, which may be useful for fitting that command into macro size limits.

**Comment by mbattersby** [comment]
Yeah WoW doesn't seem to care what you put in as the button argument... at the moment. I don't trust them not to randomly start caring though, so I do still recommend the default LeftButton argument if it fits.

**Comment by DasAoD** [comment]
`/dump GetCVar("ActionButtonUseKeyDown")`
resulted in for me:
```
Dump: value=GetCVar("ActionButtonUseKeyDown")
[1]="1"
```
So I changed the macro to `/click LM_B1 LeftButton 1` and that works.

Thank you.

**Comment by XAAAM** [comment]
Anyone got this to work with OPie? Only works if I have it on my bars :/

**Comment by mbattersby** [comment]
There is a workaround for this

https://github.com/Stanzilla/WoWUIBugs/issues/317#issuecomment-1510847497

which is implemented in 3516fc0 and 10.1.0-1. Everyone should go back to just
```
/click LM_B1
```
